app/controllers/players_controller.rb

```diff
@@ -200,7 +201,8 @@ def render_player_list_for_standings(stage)
         extended_sos: 0,
         corp_points: 0,
         runner_points: 0,
-        manual_seed: player.manual_seed
+        manual_seed: player.manual_seed,
+        side_bias: stage.format == 'single_sided_swiss' ? player.side_bias : nil
```

Comment:
This method is just for the player meeting, so it won't have side bias. Maybe the method should be renamed?

Reply:
Ah, good call.  Done.